.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: ruby
+
+rvm:
+- 2.2
+- 2.1
+- 2.0.0
+- 1.9.3
+- 1.8.7
+- jruby-19mode
+- rbx-2
```

Comment:
Hah, you might be right there... I'll look into that to see if there's a way to just do it once, but if it doesn't work, I'll just strip out the versions and leave the default.